src/Nowruz/modules/layout/containers/notifications/useNotifications.tsx

```diff
@@ -99,13 +121,18 @@ export const useNotifications = (handleClose: () => void) => {
       case 'EDUCATION_VERIFY_APPROVED':
         path = '/credentials#issued';
         break;
+      case 'DISPUTE_INITIATED':
+      case 'DISPUTE_NEW_RESPONSE':
+      case 'DISPUTE_JUROR_CONTRIBUTION_INVITED':
+        return;
```

Comment:
Based on your code, I get that clicking on 'dispute' notification won't lead to any page, just clicking on 'view details'  link redirects user to another page. Is it true based on design?

Reply:
I asked Elaine, and she said it's better to click only on the link and not whole body  for now (for copy/paste reasons)